docs/application/dotnet/tutorials/test/testing-your-app-on-gear.md

```diff
@@ -0,0 +1,72 @@
+# Test Tizen .NET Application on a Gear device
+
+To deploy Tizen .NET Application to Samsung Gallaxy app store, you need to test it on Samsung Gear devices.
```

Comment:
Suggestion:
To publish a Tizen .NET application to Samsung Galaxy Apps store, you have to test it on Samsung Gear devices. 

Please verify whether the technical meaning is altered.
We publish or deploy an app in store?

Reply:
We publish an app in store.

I apply your sentence.